installation/resources/crds/eventing/subscriptions.eventing.kyma-project.io.crd.yaml

```diff
@@ -3,6 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.11.1
+  creationTimestamp: null
```

Comment:
Is this needed?

Reply:
that's the magic of `make copy-external-crds`